src/components/bottom-bar-item/bottom-bar-item.stories.tsx

```diff
@@ -124,9 +127,11 @@ export const StyleButtonUsage: Story = {
         </ElButtonBottomBarItemContainer>
 
         <ElButtonBottomBarItemContainer onClick={action('handleClick')} aria-current={undefined} className="el-mx5">
-          <ElBottomBarItemIcon>{args?.icon}</ElBottomBarItemIcon>
-          <ElBottomBarItemLabel>{args?.children}</ElBottomBarItemLabel>
-          <ElBottomBarItemBadge role="status" />
+          <ElBottomBarItemContent>
+            <ElBottomBarItemIcon>{args?.icon}</ElBottomBarItemIcon>
+            <ElBottomBarItemBadge role="status" />
```

Comment:
**nit-pick:** I don't think `status` is an appropriate ARIA role. I'm not sure what is, but when I read [the MDN docs on the status role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/status_role), it doesn't feel quite right 🤷

Reply:
> I don't think status is an appropriate ARIA role.

ah yes, I forgot to remove it for storybook styles usage after we've agree on this [feedback](https://github.com/reapit/elements/pull/193#discussion_r1833496895)